apps/betterangels-backend/common/models.py

```diff
@@ -220,6 +221,25 @@ def get_or_create_location(cls, location_data: Dict[str, Any]) -> "Location":
         return location
 
 
+class PhoneNumber(models.Model):
```

Comment:
Probably need to follow the index pattern similar to the attachment model.  We likely want to have an index on `context_type` and `object_id`